.npmrc

```diff
@@ -1,2 +1 @@
 registry=https://registry.npmjs.org/
-package-lock=false
```

Comment:
Should we be committing the new lockfile?

Reply:
this repo is fine because it is used server-side but good catch